pkg/machine/wsl/util_windows.go

```diff
@@ -216,20 +217,23 @@ func reboot() error {
 		}
 	}
 
+	if err := addRunOnceRegistryEntry(command); err != nil {
+		return err
+	}
+
 	message := "To continue the process of enabling WSL, the system needs to reboot. " +
 		"Alternatively, you can cancel and reboot manually\n\n" +
 		"After rebooting, please wait a minute or two for podman machine to relaunch and continue installing."
 
 	if MessageBox(message, "Podman Machine", false) != 1 {
 		fmt.Println("Reboot is required to continue installation, please reboot at your convenience")
+		if err := removeRunOnceRegistryEntry(); err != nil {
```

Comment:
If you remove the entry from the registry, then `podman machine init` won't run at restart?

Reply:
Thank you.